IntegrationTests.Common/Utilities/TestResultSerializer.cs

```diff
@@ -112,6 +112,8 @@ private void SerializeWithoutSpacing (object value)
     {
       if (value == null)
         _textWriter.WriteLine ("null");
+      else if (value is DateTime dateTime && dateTime.ToLongTimeString() == "00:00:00")
+        SerializeDateTimeWithNoTime(dateTime);
       else if (value is string)
         SerializeString ((string) value);
```

Comment:
Considering NULL isn't getting special treatment, it would be OK to inline this. There's just a single line of code. I'm okay with the separate method, too.